pegasus/sites.v3/hourofcode.com/i18n/bg.yml

```diff
@@ -246,7 +246,7 @@ bg:
   signup_submit_error_message: "Възникнала е грешка. Моля проверете дали всички задължителни полета са попълнени правилно."
   signup_thank_you_header: "Благодаря ви, че регистрирахте събитие за Hour of Code."
   signup_thank_you_message: 'Ние ще ви изпратим имейл за потвърждение с информация, която да ви помогне да започнете.'
-  map_title: "\"Часът на кода\" карта"
```

Comment:
It's hacky but we could do something like this for every page title?

`title: "<%= hoc_s(:map_title).gsub(/"/,"\\\"") %>"`